animations/lib/src/misc/card_swipe.dart

```diff
@@ -112,9 +112,9 @@ class SwipeableCard extends StatefulWidget {
 
 class _SwipeableCardState extends State<SwipeableCard>
     with SingleTickerProviderStateMixin {
-  AnimationController _controller;
-  Animation<Offset> _animation;
-  double _dragStartX;
+  late AnimationController _controller;
+  late Animation<Offset> _animation;
+  double? _dragStartX;
```

Comment:
I think this can be `late double` since _dragStart is called before `_dragUpdate`

Reply:
Nice catch. You're right, it's never assigned a nullable value.